### PR TITLE
feat: pass events to window opener

### PR DESCRIPTION
**Pull request by im-adithya**
Instead of redirecting from the backend, this change opens the hub window using js and propogates events back to the opener.

If there's no opener, it simply redirects to the hub

# Full Window
https://github.com/user-attachments/assets/c6a0ebc4-1ee3-48f8-9b42-497601be3c60

# Minimized Window
https://github.com/user-attachments/assets/378613e1-768c-4133-ab39-16a2763516bf

### CODE REVIEW

File: views/apps/new.html [hunk 1]
```diff
@@ -1,5 +1,52 @@
...
+  if (!window.opener) {
+    // if there's no opener, redirect to hub url
+    document.getElementById("message").innerHTML = "Redirecting to Alby Hub..."
+    window.location.href = "{{.User.HubUrl}}/#/apps/new?{{.QueryString}}"
+  } else {
+    document.title = "Waiting for Alby Hub Connection...";
+    document.getElementById("message").innerHTML = `Please enable pop-ups and proceed in the new Alby Hub window.<br/>Keep this window open to receive updates.`
+
+    // open hub in another window
+    const hubWindow = window.open("{{.User.HubUrl}}/#/apps/new?{{.QueryString}}")
```
Comment (bumi) [inline views/apps/new.html]: can you check how we do this in bitcoin connect? - that we do it exactly the same. also with the same window size.

Comment (im-adithya) [reply views/apps/new.html]: We do it here: https://github.com/getAlby/bitcoin-connect/blob/master/src/components/connectors/bc-alby-nwc-connector.ts#L21

It comes from js-sdk: https://github.com/getAlby/js-sdk/blob/master/src/NWCClient.ts#L376-L387

So I even match the positions? Or shift some by x=10 and y=10?

Comment (bumi) [reply views/apps/new.html]: do the same thing, I'd say.

Comment (im-adithya) [reply views/apps/new.html]: Done! (check the video in PR comment to see how it works)

**Pull request merged**